core/src/input/input.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { _Story } from "../_story";
+import { Input } from "./input";
+
+export default {
+	title: "Components/Input",
+	component: Input,
+	argTypes: {
+		type: _Story.arg(null),
```

Comment:
Actually you can have this as a list of string options ("text" is the default, then "email", "password", "date", etc.)